pages/blog/asynchronously-exporting-data-to-remote-systems.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Asynchronously Exporting Data to Remote Systems"
+date: "2021-05-27"
+author: "Pedro S Lopez"
+pullQuote: "Some batch import APIs work asynchronously and don't process your requests right away."
+image: "asynchronously-exporting-data-to-remote-systems/bus-stop.jpeg"
+tags: [engineering]
+---
+
+When integrating with destinations, there are generally two main approaches made available by API providers: **single** or **batched**. With the "single" approach, one API request usually affects a single profile in the destination. The "batched" approach, which you can read more about [here](https://www.grouparoo.com/blog/batching-api-requests), allows you to affect multiple profiles in a single API request. This is usually preferred due to rate limits put in place and because they are often faster than their single counterparts when dealing with a large number of profiles.
```

Comment:
... often faster in aggregate than their single counterparts

Reply:
When integrating with [destinations](link to a page explaining what a destination is to Grouparoo)